src/Controllers/SchemaController.js

```diff
@@ -813,6 +826,146 @@ export default class SchemaController {
       });
   }
 
+  async loadSchemas(localSchemas, database: DatabaseController) {
+    const cloudSchemas = await this.getAllClasses({ clearCache: true });
+    // We do not check classes to delete, developer need to delete manually classes
+    // to avoid data loss during auto migration
+    await Promise.all(
```

Comment:
I think you should return this and lose the async.

i.e. `return Promise.all(...`